redwood-treehouse/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIKitView.kt

```diff
@@ -16,37 +16,53 @@
 package app.cash.redwood.treehouse
 
 import app.cash.redwood.protocol.widget.DiffConsumingWidget
-import kotlinx.cinterop.CValue
 import kotlinx.cinterop.ObjCAction
 import kotlinx.cinterop.cValue
-import platform.CoreGraphics.CGRect
 import platform.CoreGraphics.CGRectZero
 import platform.UIKit.UIView
 import platform.UIKit.setFrame
 import platform.UIKit.subviews
+import platform.UIKit.superview
 
 public class TreehouseUIKitView<T : Any>(
-  private val content: TreehouseView.Content<T>,
+  private val treehouseApp: TreehouseApp<T>,
 ) : TreehouseView<T> {
-  public val view: UIView = RootUiView(frame = cValue { CGRectZero })
-  private var treehouseApp: TreehouseApp<T>? = null
+  public val view: UIView = RootUiView(this)
+  private var content: TreehouseView.Content<T>? = null
 
-  // TODO(jwilson): track when this view is detached from screen
-  override val boundContent: TreehouseView.Content<T>? = content
+  override val boundContent: TreehouseView.Content<T>?
+    get() {
+      return when {
```

Comment:
`return view.superview?.let { content }`?

Reply:
Yes! I coped this style from the existing Android one so that they match, for now. There's definitely more terse ways to do this that I will refactor all of them to use in a follow-up.